style.css

```diff
@@ -66,6 +69,98 @@ form input {
     fill: var(--red);
 }
 
+
+/* main card */
+
+main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-grow: 1;
+}
+
+
+p {
+    margin: 1rem;
+}
+
+
+
+.theme {
+    background: var(--white);
+    padding: 1rem;
+    width: 100%;
+    max-width: 600px;
+    border-radius: 1rem;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.main-form {
+    margin: 1rem;
+
+}
+
+.h1 {
+    color: var(--green);
+    margin-left: 1rem;
+    margin-right: 3rem;
+}
+
+main img {
+    margin-left: 10rem;
+}
+
+.login-input {
+    background-color: rgba(195, 233, 247, 0.68);
+    padding: 0.3rem;
+    border-radius: 0.3rem;
+    border: 1px solid lightgrey;
+}
+
+.login-input--border-left {
+    background-color: rgba(195, 233, 247, 0.68);
+    padding: 0.3rem;
+    border-radius: 0.3rem 0 0 0.3rem;
+    border: 1px solid lightgrey;
```

Comment:
Those classes have a lot of duplicate code. Let's refactor like this:

1. the `login-input` class should have all the base styles. This means all the styles it has now
2. the `login-input--border-left` class should only change the specific property it needs. In our case `border-radius`.

Then, on the input you add both classes `class="login-input login-input--border-left`.